DockerHubfile

```diff
@@ -0,0 +1,32 @@
+#### Create Jar 
+FROM gradle:jdk8 as builder 
+ 
+COPY . /home/gradle/app 
+USER root
+RUN chown -R gradle:gradle /home/gradle/app
+
+USER gradle
+WORKDIR /home/gradle/app 
+ 
+RUN mkdir -p api/src/main/resources/META-INF 
+RUN echo "build.version=$(gradle -q printVersion)" > api/src/main/resources/META-INF/build-info.properties 
+RUN echo "build.number=${BUILD_NUMBER:=docker}" >> api/src/main/resources/META-INF/build-info.properties 
+RUN echo "build.commit=$(git rev-parse HEAD)" >> api/src/main/resources/META-INF/build-info.properties 
+RUN echo "build.date=$(date)" >> api/src/main/resources/META-INF/build-info.properties 
+ 
+RUN gradle installDist bootRepackage 
+ 
+#### Actual DockerFile 
+FROM openjdk:8-jre 
+ 
+MAINTAINER "HMCTS <https://github.com/hmcts>" 
+LABEL maintainer = "HMCTS <https://github.com/hmcts>" 
+ 
+WORKDIR /opt/app 
+COPY --from=builder /home/gradle/app/api/build/install/service-auth-provider-api /opt/app . 
```

Comment:
so it's already a `WORKDIR /opt/app` and both (`/opt/app` and current dir) are used here in `COPY`. is this valid?